### PR TITLE
update header component

### PR DESCRIPTION
**Pull request by jonkiky**
Pending on Bento-tool PR

### CODE REVIEW

**Review by AjayDoddapaneni — APPROVED** [review]

**Comment by skiran86** [comment]
@jonkiky Conflicts

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻👍🏻

**Comment by AjayDoddapaneni** [comment]
Can you update packge json

**Review by skiran86 — APPROVED** [review]

**Comment by skiran86** [comment]
Merge this PR after https://github.com/CBIIT/bento-icdc-frontend/pull/32

File: src/components/Header/HeaderView.js [hunk 1]
```diff
@@ -2,13 +2,20 @@ import React from 'react';
...
+};
+
 const ICDCHeader = () => (
   <>
     <Header
       logo={headerData.globalHeaderLogo}
       easter={headerData.globalHeaderImage}
       alt={headerData.globalHeaderLogoAltText}
       homeLink={headerData.globalHeaderLogoLink}
+      nihLogoImg={nihLogoImg}
```
Comment (AjayDoddapaneni) [inline src/components/Header/HeaderView.js:18]: @jonkiky  Can you rename the prop name so that we can easily understand its a styling property

**Pull request merged**